http/errors.go

```diff
@@ -9,6 +9,7 @@ import (
 	"gopkg.in/errgo.v1"
 
 	"github.com/Scalingo/go-scalingo/v6/debug"
+	scErrors "github.com/Scalingo/go-utils/errors/v2"
```

Comment:
issue: in order to match Scalingo convention

```suggestion
	errorspkg "github.com/Scalingo/go-utils/errors/v2"
```